index.html

```diff
@@ -10,6 +10,7 @@
 </head>
 
 <body>
+  <header>Developer Tools</header>
```

Comment:
Why are you using `&::before` here?. 
You can just write something like this:
<img width="145" alt="captura de pantalla 2018-06-01 a la s 12 36 05" src="https://user-images.githubusercontent.com/36163179/40852429-632df276-6598-11e8-90fd-65539d60ad01.png">

Reply:
Also there is a `<nav/>` tag that is used to wrapp links but in this case as we dont have links there is no need to use it.